src/main/java/com/example/nasa/dto/UserDTO.java

```diff
@@ -20,4 +20,6 @@
     private String email;
 
     private String photo;
+
+    private Long notificationRange;
```

Comment:
Czy to nie powinno się odnosić do FireReportDTO?

Reply:
https://trello.com/c/y28N7Ae4/65-dodanie-notificationrange-do-encji-user